packages/test-interface-criteria/src/criteria/TestOpportunityBookable.js

```diff
@@ -16,19 +13,8 @@ const doesNotSupportMinimalProposalFlow = complement(supportsMinimalProposalFlow
 const TestOpportunityBookable = createCriteria({
   name: 'TestOpportunityBookable',
   opportunityConstraints: [],
-  offerConstraints: [
-    [
-      'Does not support Minimal Proposal flow',
-      doesNotSupportMinimalProposalFlow,
-    ],
-  ],
-  testDataShape: () => ({
-    offerConstraints: {
-      'oa:openBookingFlowRequirement': openBookingFlowRequirementArrayConstraint({
-        excludesAll: ['https://openactive.io/OpenBookingApproval'],
-      }),
-    },
-  }),
+  offerConstraints: [],
+  testDataShape: () => ({}),
```

Comment:
This criteria no longer needs to include its own openBookingFlowRequirement stuff